api/turing/validation/validator.go

```diff
@@ -148,6 +149,50 @@ func checkDefaultTrafficRule(
 	}
 }
 
+func validateDefaultTrafficRuleDefaultRoute(
```

Comment:
Hey I meant move the newly added logic and merge the validation of default route against both the traffic rules and default traffic rules (my first comment): https://github.com/caraml-dev/turing/pull/240/files#r970310168

You can append everything into one array of rules and check that each rule has the default route.

Reply:
So the logic under `for ruleIdx, rule := range routerConfig.TrafficRules { ...` in the main  `validateRouterConfig` function can be simpler and we don't have to write the same kind of checks twice.

Reply:
Ah sorry! I misread your comment. I've updated the validation now, should be much cleaner, thanks for suggestion 🙏🏻

Reply:
I'll merge by EOD if there's nothing else!

Reply:
Np. I left a super nit comment. Please feel free to merge after reviewing. Nothing else from me. :)